packages/salesforce-adapter/src/fetch.ts

```diff
@@ -175,6 +176,12 @@ const getFullName = (obj: FileProperties, addNamespacePrefixToFullName?: boolean
   if (!obj.namespacePrefix) {
     return obj.fullName
   }
+
+  // Managed package names should exactly match obj.namespacePrefix
+  if (obj.fullName === obj.namespacePrefix) {
```

Comment:
Is this code relevant for metadata instances of type **InstalledPackage** only?
If so, wouldn't that be more accurate?
```suggestion
  if (obj.type === INSTALLED_PACKAGE_METADATA) {
```